.gitignore

```diff
@@ -11,6 +11,8 @@ moc_*.cpp
 ui_*.h
 qrc_*
 default/
+tests/mapreader/mapreader
+tests/staggeredrenderer/staggeredrenderer
```

Comment:
I'm personally not using qmake while developing, except for testing whether it works. And when I do, I rarely do an in-source build (Qt Creator builds outside of the source folder by default). And then still, I don't build the tests. So, there you have it. :-)